.github/workflows/cd.yml

```
@@ -243,59 +243,53 @@ jobs:
           if-no-files-found: error
           retention-days: 1
 
-  # Build for Windows with Cross
+  # Build for Windows
   build-push-windows-release-artifacts:
     permissions:
       id-token: write
       contents: write
     needs: update-release-draft
-    runs-on: ubuntu-22.04
+    runs-on: windows-2019
     defaults:
       run:
         working-directory: ./rust
     strategy:
       fail-fast: false
+      # The matrix is 1x1 to match the style of build-push-linux-release-artifacts
+      # In the future we could try to cross-compile aarch64-windows here.
       matrix:
-        arch:
-          - target: x86_64-pc-windows-msvc
-            shortname: x64
         name:
           - package: firezone-windows-client
             artifact: windows-client
     env:
-      BINARY_DEST_PATH: ${{ matrix.name.artifact }}-${{ matrix.arch.shortname }}
+      BINARY_DEST_PATH: ${{ matrix.name.artifact }}
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup-rust
         with:
           targets: x86_64-pc-windows-msvc
-      - uses: taiki-e/install-action@v2
-        with:
-          tool: cross
       - name: Build release binaries
         run: |
-          set -xe
+          cargo install tauri-cli
 
-          cross build --release -p ${{ matrix.name.package }} --target ${{ matrix.arch.target }}
+          # Tauri build already defaults to '--release'
+          cargo tauri build
 
           # Used for release artifact
-          cp target/${{ matrix.arch.target }}/release/${{ matrix.name.package }} $BINARY_DEST_PATH
+          cp target/release/${{ matrix.name.package }}.exe "${{ env.BINARY_DEST_PATH }}-x64.exe"
+
+          pwd
+          ls *.exe
+
+          # There's an MSI and NSIS installer here but their names include the version so I'll try those later.
 
-          # Used for Docker images
-          cp target/${{ matrix.arch.target }}/release/${{ matrix.name.package }} ${{ matrix.name.package }}
-          sha256sum $BINARY_DEST_PATH > $BINARY_DEST_PATH.sha256sum.txt
 
-          ls -la $BINARY_DEST_PATH
-          ls -la $BINARY_DEST_PATH.sha256sum.txt
       - name: Upload Release Assets
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          set -xe
-
           gh release upload ${{ needs.update-release-draft.outputs.tag_name }} \
```

```suggestion
          # TODO: Calculate sha256sum for this binary and attach to release
          gh release upload ${{ needs.update-release-draft.outputs.tag_name }} \
```

We tend to use `TODO`s for things that have a high degree of needing a fix later